tst/testbugfix/00065.tst

```diff
@@ -0,0 +1,5 @@
+# 2005/04/26 (SL, FL)
+gap> AClosestVectorCombinationsMatFFEVecFFECoords;
+<Operation "AClosestVectorCombinationsMatFFEVecFFECoords">
+gap> ConstituentsPolynomial;
+function( p ) ... end
```

Comment:
I know you are not cleaning up these tests right now, but here is one wher I am really confused as to what it is supposed to test, exactly...